src/Simulation/DocumentationGenerationStep/ProcessDocComments.cs

```diff
@@ -0,0 +1,228 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Quantum.QsCompiler;
+using Microsoft.Quantum.QsCompiler.DataTypes;
+using Microsoft.Quantum.QsCompiler.Documentation;
+using Microsoft.Quantum.QsCompiler.SyntaxTokens;
+using Microsoft.Quantum.QsCompiler.SyntaxTree;
+using Microsoft.Quantum.QsCompiler.Transformations.Core;
+using Newtonsoft.Json.Linq;
+
+#nullable enable
+
+namespace Microsoft.Quantum.Documentation
+{
+    public class ProcessDocComments
+    : SyntaxTreeTransformation<ProcessDocComments.TransformationState>
```

Comment:
If you don't use a transformation state, you can just use the SyntaxTreeTransformation class (no type argument).

Reply:
Sounds good. I wasn't sure if I needed a transformation state or not yet; I will likely need one for the TOC, but I can definitely remove it until I get there. Thanks!